hx_lti_initializer/static/vendors/development/video.dev.js

```diff
@@ -6309,6 +6320,10 @@ vjs.TextTrackMenuItem = vjs.MenuItem.extend({
 vjs.TextTrackMenuItem.prototype.onClick = function(){
   vjs.MenuItem.prototype.onClick.call(this);
   this.player_.showTextTrack(this.track.id_, this.track.kind());
+  if (typeof(jQuery.publish) === "function") {
```

Comment:
Hmmm see this is where it gets tricky. that video.dev.js file has existed since before I even worked on this project. I don't know in what ways it has all changed since it was first downloaded. You're right this needs to happen regardless on whether or not this version is merged or not since there are already things that exist that were added on in previous iterations. 

I'll note that in our tracker and make this a priority in upcoming fixes.